schema/deploy/shipping/views.sql

```diff
@@ -2195,6 +2196,96 @@ grant select
     to "scan-dashboard-exporter";
 
 
+create or replace view shipping.scan_demographics_v2 as
+
+    with hcov19_presence_absence as (
+        -- Collapse potentially multiple hCoV-19 results
+        select distinct on (sample_id)
+            sample.identifier as sample,
+            presence_absence_id,
+            case
+              when present is true then 'positive'
+              when present is false then 'negative'
+              when present is null then 'positive'
```

Comment:
is this a typo?

Reply:
ah wait, I forgot that Formative wants to treat inconclusive results as positive.